package-lock.json

```diff
@@ -1,4262 +1,8 @@
 {
   "name": "guanyin",
   "version": "1.4.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Why was this downgraded?

Reply:
Fixed now, I made sure it was created using Node 18